limit.go

```diff
@@ -127,6 +127,8 @@ func (l *BaseLimit) Apply(l2 BaseLimit) {
 }
 
 // BaseLimitIncrease is the increase per GB of system memory.
+// Memory is in bytes.  Values greater than 1<<30 likely don't make sense.
+// FDFraction is expected to be >= 0 and <= 1.
```

Comment:
Can you move these 2 comments just above the respective fields in the struct. IDEs will then be able to show that documentation when setting that field.

Reply:
1<<30 is 1GIB. Values above this make sense (fyi)

Reply:
oh sorry, this is in the increase struct. Yes values above 1<<30 don't make sense because you're saying you want to allow over 1GiB for every 1GiB allowed